packages/discord.js/src/structures/ApplicationCommand.js

```diff
@@ -127,11 +127,11 @@ class ApplicationCommand extends Base {
   /**
    * Data for creating or editing an application command.
    * @typedef {Object} ApplicationCommandData
-   * @property {string} name The name of the command
-   * @property {string} description The description of the command
-   * @property {ApplicationCommandType} [type] The type of the command
+   * @property {string} name The name of the command, must be in all lowercase if type is CHAT_INPUT
+   * @property {string} description The description of the command, if type is CHAT_INPUT
+   * @property {ApplicationCommandType} [type=ApplicationCommandType.ChatInput] The type of the command
    * @property {ApplicationCommandOptionData[]} [options] Options for the command
-   * @property {boolean} [defaultPermission] Whether the command is enabled by default when the app is added to a guild
+   * @property {boolean} [defaultPermission=true] Whether the command is enabled by default when the app is added to a guild
```

Comment:
Lint is failing here because the line is too long.
```suggestion
   * @property {boolean} [defaultPermission=true] Whether the command is enabled by default when the app is added to a
   * guild
```